lib/diagnostic.rb

```diff
@@ -6,37 +6,47 @@
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = 'ruby lib/example.rb'
 
 # Using the REPL
 #
 # In a Ruby string, write the command you would use to enter the REPL we
 # prefer over `irb`
 
-Diagnostic.repl = 'your response here'
+Diagnostic.repl = 'pry'
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+Diagnostic.variable = 'variable = "star wars episode viii"'
 
 # String interpolation in Ruby
 #
 # Use string interpolation to complete the sentence below. Replace "ANSWER" with
 # the correct code.
-
+answer = 'ANSWER'
 character = 'Jar Jar Binks'
+character.prepend('is really a sith lord')
+answer.delete!('ANSWER')
+
+
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = "'character' + 'is really a Sith Lord.'"
 
 # Define a predicate method in Ruby
 #
 # Define a method that checks whether a number passed in is odd. Name it
 # conventionally.
 
 # define your method here
+ary = [2, 3, 5, 7, 10, 5]
+arra
+ary_odd = ary.select { |e|e.odd }
+
+
+
```

Comment:
we were looking for a new method defined conventionally for a predicate, such as:

```
def num_odd? (n)
  # something with n
end
```